src/components/Reports/Subscribers/Subscribers.js

```diff
@@ -105,6 +105,15 @@ const Subscribers = ({ dependencies: { dopplerApiClient } }) => {
               <SubscriberInfo subscriber={state.subscriber} />
               <Component subscriber={state.subscriber} />
             </div>
+            {/*
+            TODO: Make the method in the api-client
+              section === 'gdpr' && 
+              <div className="dp-cta-links">
+                <a href="#"><span className="ms-icon icon-download"></span>
+                  {" "}<FormattedMessage id="subscriber_history.download_history" />
+                </a>
+              </div>
+            */}
```

Comment:
What do you think about removing all related CSV download code out of this PR?

Reply:
I will fix it in the next commit :)

Reply:
Done! resolving conversation...